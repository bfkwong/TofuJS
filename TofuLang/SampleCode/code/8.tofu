blueprint for Cat {
    name;
    gender; 
    weight;
    age;
    breed;
    spayed;
    other;
}

cat = make Cat;
cat.name = "Tofu";
cat.gender = "female";
cat.weight = 10;
cat.age = 1; 
cat.breed = "Domestic Shorthair";
cat.spayed = true;

print(cat.name);
print(cat.gender);
print(cat.weight);
print(cat.age);
print(cat.breed);
print(cat.spayed);
print(cat.other);