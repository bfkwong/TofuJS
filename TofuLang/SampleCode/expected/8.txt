Tofu
female
10
1
Domestic Shorthair
true
undefined
